src/radical/pilot/agent/scheduler/continuous.py

```diff
@@ -236,6 +236,11 @@ def _alloc_node(self, node, requested_cores, requested_gpus,
 
     # --------------------------------------------------------------------------
     #
+
+    '''
+    for each process (containing X threads) generate a process map that requests 1 core per thread.
+    core maps returns a list of lists of process maps
+    '''
```

Comment:
Two tidbits: first, docstring comments need to be *inside* the documented method, so that they get picked up by, for example, online doc generators.  Second, please add a sentence about GPUs.  Otherwise that level of detail is very suitable for a short, internal helper method - thanks!